webrecorder/templates/collection_info.html

```diff
@@ -26,6 +26,8 @@
 <script>
     var can_admin = {{ 'true' if can_admin() else 'false' }};
     var can_write = {{ 'true' if can_write() else 'false' }};
+
+    var init_selected_recs = {{ rec_list }};
```

Comment:
the pattern of assigning vars in script tags in the template is not optimal, we should avoid introducing more new code like that.  from this quick look it seems like on page load you could render the recording elements with the  card-selected in the template and then reuse the same code paths to update bookmarks table.